pyproject.toml

```diff
@@ -50,9 +50,9 @@ build_command = "pip install poetry && poetry build"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-click = ">= 7, <9"
-toml = "^0.10.2"
-rich = "^12.2.0"
+click = ">=7, <9"
```

Comment:
Click is still constrained here to be `<9`, which is probably not too bad, as Click doesn't bump major as often as rich, but just wanted to make sure this was intentional.

Reply:
but besides that LGTM! Thanks a bunch!

Reply:
True! Fixed